package.json

```diff
@@ -26,9 +27,11 @@
     "lint:js:fix": "npx eslint . --fix",
     "prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
     "release": "multi-semantic-release --ignore-packages=packages/**",
-    "test": "turbo run test",
-    "tscheck": "turbo run prisma-generate && turbo run tscheck --parallel --filter=!@pins/api",
-    "web": "npm run dev --workspace=@pins/web"
+    "test": "turbo run test --filter=./apps/*",
```

Comment:
We still want to test packages. Do we need a filter on this one? If we do, let's do `!./appeals/*`?